careers/application-process/index.html

```diff
@@ -0,0 +1,292 @@
+{% extends "layout-side-nav.html" %}
+{% import "_vars-careers.html" as vars with context %}
+{%- set active_nav_id = 'application-process' -%}
+
+{% block title -%}
+    Application Process
+{%- endblock %}
+
+{% block desc -%}
+
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
+{%- endblock %}
+
+{% block hero -%}
+    <section class="content_hero hero hero__bureau">
+        {% from "macros.html" import YouTube %}
+        {% call YouTube({
+            'video_id' : 'XCbZnTIeTOY',
+            'image_res' : 'max',
+            'image_url': '../../static/img/applying-video-screenshot.png'
+            })
+        %}
+        <div class="hero_wrapper hide-on_yt-playing">
+            <div class="hero_card hero_card__bureau">
+                <article class="hero_preview">
+                    <div class="summary">
+                        <header class="summary_header">
+                            <h1 class="summary_heading">CFPB Hiring Video</h1>
+                        </header>
+                        <p class="summary_text">
+                            {# TODO: This is placeholder content #}
+                            We are a government agency created after the 2008 financial crisis.
+                            We protect American consumers. Allow us to introduce ourselves.
+                        </p>
+                        <footer class="summary_footer">
+                            <a class="jump-link jump-link__right" href="/history/">
+                                <span class="icon-link_text">See the history of the CFPB</span>
+                            </a>
+                        </footer>
+                    </div>
+                </article>
+            </div>
+        </div>
+        {% endcall %}
+    </section>
+{%- endblock %}
+
+{% block content_main %}
+    <section class="block block__flush-top block__sub">
+            <h1>Job Application Process</h1>
+            <p class="h3">
+                Applying for a federal job can take some time and patience.
+                It is a process that prioritizes both fairness to all applicants
+                and getting the best qualified candidates.
+                Watch the video above (or read the transcript)
+                for an overview of how it works.
+            </p>
+            <p class="short-desc">
+                Please note that to ensure this process is followed,
+                we can’t take applications directly.
+                You must submit your application in one of the ways
+                described on USAJobs.
+                Here are some things to keep in mind:
+            </p>
+    </section>
+
+    <section class="job-applications-interfaces">
+        <h2>Different interfaces you may have to use</h2>
+        <div class="expandable
```

Comment:
Should the expandables be closed by default? I'd imagine most people would rather scroll than close them one by one.